scripts/run-live-build.sh

```diff
@@ -85,41 +122,18 @@ if ! [[ -n "$CI" && -n "$TRAVIS" ]]; then
 	# about these details.
 	#
 
-	BUCKET="snapshot-de-images"
-	JENKINSID="jenkins-ops"
-
-	if [[ -z "$AWS_S3_PREFIX_VIRTUALIZATION" ]]; then
-		URI="s3://$BUCKET/builds/$JENKINSID/dlpx-app-gate/"
-		URI+="projects/dx4linux/build-package/post-push/latest"
-
-		aws s3 cp "$URI" .
-		AWS_S3_PREFIX_VIRTUALIZATION=$(cat latest)
-		export AWS_S3_PREFIX_VIRTUALIZATION
-		export AWS_S3_BUCKET="snapshot-de-images"
-		rm -f latest
-	fi
+	AWS_S3_URI_VIRTUALIZATION=$(get_s3_uri "$AWS_S3_PREFIX_VIRTUALIZATION" \
```

Comment:
I don't really like how we're now using a variable named "prefix" to potentially specify the full URI, not a prefix.

I think I'd rather do something like:
```
if [[ -z "$AWS_S3_URI_VIRTUALIZATION" ]]; then
    export AWS_S3_URI_VIRTUALIZATION=$(...)
fi
```

Then we expose the `AWS_S3_URI_VIRTUALIZATION` variable, and expect consumers to actually populate it with a URI.. rather than mixing the terms "prefix" and "URI", which can be confusing for the reader and consumers of this script.

Reply:
Sorry, I forgot to mention that devops is working to refactor the jenkins job.
The goal is indeed to rename PREFIX to URI once that is done. I think for now we should be able to live with prefix accepting URIs (note that prefixes are still accepted).

In my first implementation I actually did create AWS_S3_BUCKET_VIRTUALIZATION and AWS_S3_BUCKET_MASKING but realized that is not useful, as we only need the full URI, and it simplifies things.